kubeportal/api/views/info.py

```diff
@@ -25,8 +25,7 @@ class InfoDetailView(GenericAPIView):
              'k8s_pod_count': api.get_number_of_pods,
              'k8s_volume_count': api.get_number_of_volumes,
              'portal_user_count': get_user_count,
-             'portal_version': get_kubeportal_version,
-             'k8s_cluster_name': get_cluster_name,
```

Comment:
I know that having the cluster_name displayed in the info section is redundant, but isn't it also a valid info to display?

Reply:
I came to the conclusion that "cluster name" and "portal branding" is semantically something different, meanwhile. Kubeportal is not always about K8S access. I therefore vote for leaving out the idea of a "cluster name" for the moment.